### DISCUSSION TITLE
pixelformat=rgb it does not detect faces

### DISCUSSION THREAD

**Discussion by GabrielYuvone** [luicfrr/react-native-vision-camera-face-detector#82] [Q&A] [answered]
hello, when pixelformat is rgb dont detetcted faces but in yuv yes. any idea?

![imagen](https://github.com/user-attachments/assets/520d6ca3-2be0-42ab-8b64-edf8f12f290a)
![imagen](https://github.com/user-attachments/assets/aa2af421-d8b6-45f2-8bed-848ee089df41)

**Comment by luicfrr** [comment]
Pixel format should always be yuv.

**Comment by GabrielYuvone** [comment]
hi, thanks for respond, im working taking the example of https://github.com/mrousavy/FaceBlurApp/blob/main/app/App.tsx and here the pixelFormat is rgb, changing this to yuv the face is detected but the camera is very slowly the respons

**Reply by luicfrr** [reply]
Take a look [here](https://developers.google.com/ml-kit/vision/face-detection/android#real-time_face_detection):
> If you use the Camera2 API, capture images in ImageFormat.YUV_420_888 format. If you use the older Camera API, capture images in ImageFormat.NV21 format.

**Reply by luicfrr** [reply]
I can't say if face detection works with rgb, I never used it this way so use it at your own risk.

**Reply by luicfrr** [reply]
Also, take a look into VC's docs [here](https://react-native-vision-camera.com/docs/guides/pixel-formats#what-are-pixel-formats).

**Reply by GabrielYuvone** [reply, marked as answer]
yes, much better, I had contourMode: "all" and
 landmarkMode: "all", at once, thanks! (pixelformat yuv)
